woocommerce/woocommerce-admin
# Add Dependency Extraction Webpack Plugin

Fixes https://github.com/woocommerce/woocommerce-admin/issues/5647

Changes our script dependencies to use our own Dependency Extraction Webpack Plugin. This PR updates the main script, packages, and wc-admin scripts.

### Detailed test instructions:

1. `npm start`
1. Make sure the app builds and no fatal errors occur as you visit screens and pages.
2. Check the console for errors relating to js files not being found.
3. Visit the coupons page to ensure that wc-admin-scripts are also working.

### Bonus Testing

1. Add a new dependency to a package. ie, add `import { compose } from '@wordpress/compose';` to any package that would have that dependency, such as `@woocommerce/currency`. Note, you may need to log it out so it gets used, otherwise webpack may disregard it.
2. Confirm the presence of `wp-compose` in `dist/currency/index.min.assets.php`

### Changelog Note:

- Dev: Use Woo Dependency Extraction Webpack Plugin to gather dependencies
